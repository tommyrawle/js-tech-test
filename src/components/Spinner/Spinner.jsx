import React from 'react';
import SpinnerGif from './Spinner.gif';
import { SpinnerWrapper, SpinnerImage } from './Spinner.styles.jsx';

const Spinner = () => {
  return (
    <SpinnerWrapper>
      <SpinnerImage src={SpinnerGif} alt="loading indicator" />
    </SpinnerWrapper>
  );
};

export default Spinner;
