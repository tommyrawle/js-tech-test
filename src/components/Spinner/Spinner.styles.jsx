import styled from 'styled-components';

export const SpinnerWrapper = styled.div`
  text-align: center;
`;
export const SpinnerImage = styled.img`
  width: 100px;
  margin: 4rem;
`;
