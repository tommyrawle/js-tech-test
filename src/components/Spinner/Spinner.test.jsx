import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './Spinner.jsx';

it('expect to render Spinner component', () => {
  expect(shallow(<Spinner />)).toMatchSnapshot();
});
