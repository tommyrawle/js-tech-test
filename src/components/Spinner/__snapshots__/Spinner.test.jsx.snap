// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Spinner component 1`] = `
<styled.div>
  <styled.img
    alt="loading indicator"
    src={null}
  />
</styled.div>
`;
